src/fixes.cpp

```diff
@@ -50,6 +50,9 @@ namespace fixes
         if (config::fixVerticalLookSensitivity)
             PatchVerticalLookSensitivity();
 
+		if (config::fixAnimationLoadSignedCrash)
+			PatchAnimationLoadSignedCrash();
```

Comment:
Hey, guess what? Another indentation mismatch.

I guess at this point, you're going to know to look for more indentation mismatches on your own.